bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -111,6 +129,7 @@ export class DataTableView extends WidgetView {
 
   protected _in_selection_update = false
   protected _warned_not_reorderable = false
+  protected _width: number | null
```

Comment:
This should be either `protected _width?: number`, `protected _width: number | undefined` or `protected _width: number | null = null`. The current version is a minor inconsistency, but still visible. TypeScript can warn about lack of initialization, but we can't enable this feature at this point.